### ISSUE TITLE
tts idea

### ISSUE THREAD

**Issue by KalvinThien** [MJP1998/Auto_short_generator#2]
In terms of TTS voice, I think you should give it a try edge_tts
It's free, and supports 80 different voices.

**Comment by MJP1998** [comment]
Thank you so much for the suggestion! 
I tried edge_tts, tortoise-tts, bark, and I'm currently trying coqui.ai but as of my experiments the voice always comes up as robotic/not natural/wrong when using it on long texts. I'm trying to find one with a similar quality as ElevenLabs and it's difficult for now.

**Comment by KalvinThien** [comment]
If so, I think we'll add a little trick here:
ElevenLabs allows you to use 10,000 characters / 1 month for free, and is not blocked if you use multiple accounts.
Why not register 10-20 accounts
At this time, each month there will be 100,000 - 200,000 characters to use.
Just add a condition to check that if api 1 runs out, switch to the next api to use.

It takes a bit of time, but it will have a high quality voice, and the cost of buying an email to register is not expensive, ($1 = 10 accounts).

**Comment by MJP1998** [comment]
Yes!! I'm using that same technic creating accounts on gmail for free, but would feel better to have an open-source one so it's easier to use and tweak the pipeline I guess, will try more whenever I get the time!
